src/Api/ProductModelApi.php

```diff
@@ -35,4 +37,20 @@ public function get($code)
     {
         return $this->resourceClient->getResource(static::PRODUCT_MODEL_URI, [$code]);
     }
+
+    /**
+     * Available from Akeneo PIM 2.0.
```

Comment:
from => since

Reply:
From is better here I think

Reply:
Discussed with Nono and Ludo: `since` seems better